kilo.go

```diff
@@ -530,6 +531,16 @@ func editorFind() {
 func editorOpen(filepath string) {
 	file, err := os.Open(filepath)
```

Comment:
Wouldn't it be better if we first checked if the file exists? 

Also, let's create it after the first Ctrl + s is invoked. Otherwise, this will create files even if the user didn't intend to make it. 

What I am saying is, let's keep track of the filename the user mentioned in case the file isn't found. Then, when the user says hey, I would like to save with Ctrl + s, rather than asking for the name of the file like right now, just use the provided name.

Reply:
Updated. Let me know if this looks fine.